keps/sig-network/3726-standard-application-protocols/README.md

```diff
@@ -168,21 +166,18 @@ type ServicePort struct {
   ... 
   ...
 
-  // Used as a hint for implementations to
-  // configure the protocol used between the
-  // implementation and the application it exposes.
-  // This field follows standard Kubernetes label syntax.
-  // Valid values are either:
-  //
-  // * Un-prefixed protocol names - reserved for IANA standard service names (as per
-  // RFC-6335 and https://www.iana.org/assignments/service-names).
-  //
-  // * Kubernetes standard names:
-  //   * 'k8s.io/http2' - http2 over cleartext, aka 'h2c'. https://www.rfc-editor.org/rfc/rfc7540
-  //   * 'k8s.io/grpc' - grpc traffic - see https://github.com/grpc/grpc/blob/v1.51.1/doc/PROTOCOL-HTTP2.md
-  //   * 'k8s.io/tcp' - plain tcp traffic
-  //
-  // * Other protocols should use prefixed names such as
+  // The application protocol for this port.
+	// This is used as a hint for implementations to offer richer behavior for protocols that they understand.
+	// This field follows standard Kubernetes label syntax.
+	// Valid values are either:
+	//
+	// * Un-prefixed protocol names - reserved for IANA standard service names (as per
+	// RFC-6335 and https://www.iana.org/assignments/service-names).
+	//
+	// * Kubernetes-defined prefixed names:
+	//   * 'kubernetes.io/h2c' - HTTP/2 over cleartext as described in https://www.rfc-editor.org/rfc/rfc7540
+	//
+	// * Other protocols should use implementation-defined prefixed names such as
```

Comment:
Nit: Indentation looks inconsistent here.

Reply:
where?

`kubernetes.io/h2c` is purposely under "Kubernetes-defined prefixed names:"

Reply:
Line 169 is using spaces for indentation and the lines below are using tabs.

Reply:
converted everything to spaces